components/time-picker/__tests__/index.test.js

```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import { mount } from 'enzyme';
 import moment from 'moment';
+import RCPicker from 'rc-picker';
```

Comment:
Please add relevant test cases for `antd/components/time-picker` instead of `rc-picker`.

Reply:
Removed